src/core/EdFi.DataManagementService.Core/ApiSchema/Extensions/JsonHelperExtensions.cs

```diff
@@ -3,14 +3,15 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-using Microsoft.Extensions.Logging;
+using System.Text.Json;
 using System.Text.Json.Nodes;
+using System.Text.RegularExpressions;
 using Json.Path;
-using System.Text.Json;
+using Microsoft.Extensions.Logging;
 
 namespace EdFi.DataManagementService.Core.ApiSchema.Extensions;
 
-internal static class JsonHelperExtensions
+internal static partial class JsonHelperExtensions
```

Comment:
Why a partial class? just curious

Reply:
I found the answer for myself: https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-source-generators?pivots=dotnet-8-0

Great new feature to use!

Maybe we should use that in `DocumentValidator.cs` as well? And in `StepDefinitions.cs`?